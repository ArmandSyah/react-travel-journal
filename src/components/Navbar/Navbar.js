import React from "react";  

export default function Navbar(){
    return (
        <nav>
            <i className="fa-solid fa-earth-americas fa-2x"></i> <span className="navbar--title">my travel journal.</span>
        </nav>
    )
}