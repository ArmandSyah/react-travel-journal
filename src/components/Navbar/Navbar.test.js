import React from "react";
import { render, cleanup } from "@testing-library/react";

import Navbar from './Navbar';

afterEach(cleanup);

it('renders properly', async () => {
    const { getByText } = render(<Navbar />);
    expect(getByText('my travel journal.')).toBeInTheDocument();  
});

it('renders', async () => {
    const { asFragment } = render(<Navbar />)
    expect(asFragment()).toMatchSnapshot();
});