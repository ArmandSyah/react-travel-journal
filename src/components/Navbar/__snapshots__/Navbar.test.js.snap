// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<DocumentFragment>
  <nav>
    <i
      class="fa-solid fa-earth-americas fa-2x"
    />
     
    <span
      class="navbar--title"
    >
      my travel journal.
    </span>
  </nav>
</DocumentFragment>
`;
